system/jlib/jevent.cpp

```diff
@@ -100,7 +100,7 @@ struct EventInformation
 #define ATTR_HEADER           EvAttrEventTimestamp, EvAttrEventTraceId, EvAttrEventThreadId, EvAttrEventStackTrace
 #define INDEX_HEADER          ATTR_HEADER, EvAttrFileId, EvAttrFileOffset, EvAttrNodeKind
 #define INDEXLOOKUP_ATTRS     INDEX_HEADER, EvAttrInCache, EvAttrExpandedSize
-#define INDEXLOAD_ATTRS       INDEX_HEADER, EvAttrExpandedSize, EvAttrElapsedTime, EvAttrReadTime
+#define INDEXLOAD_ATTRS       INDEX_HEADER, EvAttrExpandedSize, EvAttrExpandTime, EvAttrReadTime
```

Comment:
The reader could be changed to compute the elapsed time, meaning the expand time could be an additional attribute instead of a replacement. Maybe a separate Jira could address it if it seems reasonable.

Reply:
I would keep only expand for the moment.  If it is calculated it should be by something right at the end of the chain.